### PR TITLE
chore(ui): Import consistently from lodash

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

Recent work to delete unused dependencies reminded me of an inconsistency that the team had discussed, and then forgotten. I became curious whether rewrite named imports as default imports from lodash subfolder affects either bundle size or yarn.lock file.

1. In ui/apps/platform folder:
    ```sh
    find src \( -name '*.js' -o -name '*.ts' -o -name '*.tsx' \) -exec fgrep -h "'lodash" \{\} >> somewhere/lodash.txt \;
    ```
2. Find in File `'lodash'` 6 named out of 150 total
    * `import { isEmpty } from 'lodash';`
    * `import { debounce } from 'lodash';`
    * `import { uniq } from 'lodash';`
    * `import { uniq } from 'lodash';`
    * `import { uniqBy } from 'lodash';`
    * `import { sortBy } from 'lodash';`

Although I do not have a preference between named import from `'lodash'` versus default import from `'lodash/whatever'` a small number of exceptions to a pattern sometimes defeats my old school style to search the code.

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

TL;DR You don’t know unless you measure.

1. `yarn lint` in ui
2. `yarn` in ui
3. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js -108 = 4231761 - 4231869
        total -76834 = 10235754 - 10312588
    * `ls -al build/static/js/*.js | wc`
        files -1 = 79 - 80
4. `yarn start` in ui

### Manual testing

1. Visit /main/dashboard and while there analyze page with aXe DevTools to assess accessibility (pardon pun)

2. Continue to visit every link in left navigation and ditto

### CODE REVIEW

**Comment by bradr5** [comment]
Nice and thorough. Was curious if you either missed an import in `/ui/apps/platform/cypress/helpers/scopeSelectors.js` or just decided it could be an exception to the rule?

The import in question:
```javascript
import { isString, isPlainObject, isFunction, mapValues } from 'lodash';
```

**Review by bradr5 — APPROVED** [review]

**Comment by pedrottimark** [comment]
Oh, good eyes! I forgot to search in the cypress folder.
